src/TypeScriptGenerator.ts

```diff
@@ -776,17 +778,16 @@ function makeRawResponseProp(
         schema,
         [Array.from(nullthrows(nodeSelections).values())],
         state,
-        schema.isAbstractType(nodeType) ? null : nodeType.name
+        // TODO schema.isWrapper - when update @types/relay-compiler to v8 remove ts-ignore
+        // @ts-ignore
```

Comment:
Can you make this update first?

Reply:
Yes. I started at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41740
When its ready I will ping you

Reply:
Great, just merged it 👌